docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,810 @@ thor:
+          uses a sample OpenSSL config file found in the HPCC-Platform
+          repository.</para>
 
-          <listitem>
-            <para><emphasis role="bold">public</emphasis>: For external-facing
-            services</para>
-          </listitem>
+          <programlisting>openssl req -x509 -newkey rsa:2048 -nodes -keyout ca.key -sha256 -days 1825 \
+            -out ca.crt -config helm/examples/certmanager/ca-req.cfg</programlisting>
 
-          <listitem>
-            <para><emphasis role="bold">remote</emphasis>: For remote client
-            connections</para>
-          </listitem>
+          <para>The root certificate needs to be added as a Kubernetes secret
+          in order to be accessible to cert-manager. The secret name matches
+          the default name used in the local issuer configuration in
+          values.yaml.</para>
 
-          <listitem>
-            <para><emphasis role="bold">signing</emphasis>: For code signing
-            certificates</para>
-          </listitem>
-        </itemizedlist>
+          <programlisting>kubectl create secret tls hpcc-local-issuer-key-pair --cert=ca.crt --key=ca.key</programlisting>
+
+          <para>Repeat the root certificate creation process to create the CA
+          secret for the code signing issuer. This is needed to run the ECL
+          testing script at the end to show the example is working.</para>
+
+          <programlisting>openssl req -x509 -newkey rsa:2048 -nodes -keyout signingca.key -sha256 \
+            -days 1825 -out signingca.crt -config examples/certmanager/ca-req.cfg</programlisting>
+
+          <para>Create a Kubernetes TLS secret from the generated signing root
+          certificate and private key</para>
+
+          <programlisting>kubectl create secret tls hpcc-signing-issuer-key-pair --cert=signingca.crt --key=signingca.key</programlisting>
+
+          <para>Install the HPCC Helm chart with the "--set
+          certificates.enabled" option set to true.</para>
+
+          <programlisting>helm install myhpcc hpcc/hpcc --set certificates.enabled=true</programlisting>
+
+          <para>The cluster ESPs are now using TLS both locally and publicly.
+          Verify it is working by running an ECL job that requires using
+          mutual TLS (using local client certificate):</para>
+
+          <programlisting>ecl run --ssl hthor helm/examples/certmanager/localhttpcall.ecl</programlisting>
+
+          <para>Note that for the HTTPCALL in our ECL example the url now
+          starts with <literal>mtls:</literal> this tells HTTPCALL/SOAPCALL to
+          use mutual TLS, using the local client certificate, and to verify
+          the server using the local certificate authority certificate.</para>
 
-        <sect4 id="Remote_Issuer_Configuration">
-          <title>Remote Issuer Configuration</title>
+          <para>You should see a result similar to this, possibly with some
```

Comment:
Thinking about removing extra text that doesn't give much value: If it is in our style to do so, remove the reference to the 'additional headers' and just "snip out" (with ellipses?) the whole header section.